app/views/rent_mailer/new_rent.html.erb

```diff
@@ -4,9 +4,9 @@
       <thead>
         <tr>
           <th>ID</th>
-          <th>Book</th>
-          <th>From</th>
-          <th>To</th>
+          <th><%= t('book') %></th>
```

Comment:
We saw that this translatings really works, but we can order this better.
For the `book` translation, we can make the file for book translations like in these links: 

here for the "automatic" `book` model translations: http://guides.rubyonrails.org/i18n.html#translations-for-active-record-models

And here how we should organize the files: http://guides.rubyonrails.org/i18n.html#organization-of-locale-files